### PR TITLE
contrib: systemd: fix ocid path

### PR DESCRIPTION
**Pull request by runcom**
Signed-off-by: Antonio Murdaca <runcom@redhat.com>

### CODE REVIEW

**Comment by runcom** [comment]
@sameo @mrunalp PTAL

File: contrib/systemd/ocid.service [hunk 1]
```diff
@@ -8,7 +8,7 @@ Type=notify
 EnvironmentFile=-/etc/sysconfig/ocid-storage
 EnvironmentFile=-/etc/sysconfig/ocid-network
 Environment=GOTRACEBACK=crash
-ExecStart=/usr/bin/ocid \
+ExecStart=/usr/local/bin/ocid \
```
Comment (mrunalp) [inline contrib/systemd/ocid.service]: For a sample systemd service, I think we should retain /usr/bin/ocid as that's where the distributions will package / install it at.

Comment (runcom) [inline contrib/systemd/ocid.service]: right, but we provide `make install.systemd` target as well and running `systemctl start ocid` fails w/o this patch and it seems weird

Comment (runcom) [inline contrib/systemd/ocid.service]: distros can of course run `sed` and substitute the path as they wish I guess

Comment (mrunalp) [inline contrib/systemd/ocid.service]: Yeah, it's fine. It should be simple enough to modify this.

**Comment by mrunalp** [comment]
LGTM

File: contrib/systemd/ocid.service (hunk 1 above)
Comment (TomSweeneyRedHat) [inline contrib/systemd/ocid.service]: Is there any tweaks necessary to documentation for this change, even if it's only a suggested heads up kind of thing?

**Pull request merged**